core/connection/stubs_test.go

```diff
@@ -231,6 +209,8 @@ func (foc *connectionMock) StateCallback(callback func(state fakeState)) {
 	foc.stateCallback = callback
 }
 
+const fakeDialogLog = "[fake dialog] "
```

Comment:
Do we really need to have a log in tests?

Reply:
Makes it easier to catch issues in e2e tests.